src/vcml/debugging/gdbserver.cpp

```diff
@@ -60,15 +61,17 @@ void gdbserver::update_status(gdb_status status) {
 
     switch ((m_status = status)) {
     case GDB_STOPPED:
+        if (prev_status != GDB_STOPPED) {
+            m_c_target = find_target(pid, tid);
+            m_g_target = m_c_target;
+            if (!m_c_target)
+                VCML_ERROR("stopped because of unknown target");
```

Comment:
Can we handle this without crashing the entire simulation? Maybe log an error and just change into running state?